src/manipulation/wrapMap.js

```diff
@@ -3,12 +3,11 @@ define( function() {
 // We have to close these tags to support XHTML (#13200)
 var wrapMap = {
 
-	// Support: IE9
```

Comment:
I reverted changes in this file as I didn't have time to research that properly.